src/main/java/tictactoe/Main.java

```diff
@@ -0,0 +1,8 @@
+package tictactoe;
+
+public class Main {
+
+    public static void main(String[] args) {
+        System.out.println("I don't know what should be here, so I place here \"What a wonderful world\"");
```

Comment:
Może main by cos robil poza wypisywaniem info? Bo przez to nie mozna sobie zagrac :(((

Reply:
Może potem ;D mam inne priorytety :D